### ISSUE TITLE
500 Internal Server Error Server got itself in trouble Alexa Media Player

### ISSUE THREAD

**Issue by matteo289** [alandtse/alexa_media_player#2167]
**IMPORTANT: Please search the issues, including closed issues, and the [FAQ](https://github.com/alandtse/alexa_media_player/wiki/FAQ) before opening a new issue. The template is mandatory; failure to use it will result in issue closure.**

**Describe the bug**
HI
I also have the same error. If I install Alexa Media Player, both with public IP address and local IP address, I get the error

500 Internal Server Error Server got itself in trouble

How can it be solved?

Core --> 2024.1.2
Supervisor --> 2023.12.0
Alexa --> 4.9.0

Thank you

**To Reproduce**

**Expected behavior**

**Screenshots**

**System details**

- Home-assistant (version):
- alexa_media (version from `const.py` or HA startup):
- alexapy (version from `pip show alexapy` or HA startup):
- Amazon [2FA is enabled](https://github.com/alandtse/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account) (y/n). :

**Logs**
Please provide [logs](https://github.com/alandtse/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component).

**Additional context**

**Comment by ThijmenDam** [comment]
I have this issue as well. @ATVTourismus what do you mean by "fully"? I have enabled 2FA already (using a phone and an authenticator app).

**Comment by matteo289** [comment]
thanks for your answer.  However, am I now forced to activate 2FA in my
Amazon account to then be able to install Alexa media player?  when I
install the integration, I'm asked for a 6-digit code in addition to my
username and password.  should I write something or not?  Thank you

Il sab 13 gen 2024, 18:15 ATVTourismus ***@***.***> ha
scritto:

**Comment by matteo289** [comment]
I meant to say this.  when I install the Amazon Alexa integration, I have
to enter: username, password, website (Amazon.it).  when the site asks me
to log in to Amazon, I have to type in my credentials again.  in this case
I must then enter the opt code or code from the app chosen by me.  I don't
have to insert any opt code in your integration right?

Il sab 13 gen 2024, 19:01 ATVTourismus ***@***.***> ha
scritto:

**Comment by TICHIEM** [comment]
thank @ATVTourismus activating fully worked for me

**Comment by welle** [comment]
Same issue here. :(

**Comment by alandtse** [comment]
Dupe of #2138

**Comment by positivohildo** [comment]
Hello everyone,

I have been struggling with an issue for over 5 months now and I hope someone can help. Every time I try to configure the Alexa Media Player, I encounter the following error message:

"500 Internal Server Error - Server got itself in trouble."

I have tried several times to resolve this issue, but I haven't had any success. I have followed the installation instructions step-by-step, and even tried reconfiguring it multiple times, but the problem persists.

Has anyone experienced a similar issue or knows what might be causing this error? Any guidance or suggestions would be greatly appreciated. I'm eager to get this working as it's an important part of my smart home setup.

Thank you in advance for your help!

Best regards,
Hildo
